api/services/partnersData.js

```diff
@@ -12,6 +12,9 @@
  *      copy: 'confirmation copy'
  *    },
  *    campaignKey: 'Mobile Commons partner opt-in path key'
+ *    campaignKeyBeta: 'Optional key for signing up beta users',
+ *     referralCount: Number controling the amount of referral inputs to display,
```

Comment:
Also I think we could just name this `betaCount`. It'll be consistent with the way you're using it in other parts of your code. And in this repo and elsewhere, `referralCount` tends to mean the number of referrals a user currently has.